core/primitives/src/views.rs

```diff
@@ -648,7 +648,7 @@ pub struct ChunkHeaderView {
     pub encoded_length: u64,
     pub height_created: BlockHeight,
     pub height_included: BlockHeight,
-    pub shard_id: ShardId,
+    pub shard_ord: ShardOrd,
```

Comment:
Looks like you removed the old field directly here, which is not desirable.

Reply:
hmm this changeset looks like the changeset I committed yesterday, not sure why, I thought I deleted that one. Let me try to re-submit